### PR TITLE
using default message validators for all nodes; setting required-veri…

### PR DESCRIPTION
**Pull request by vishalchangrani**
…fication-seal-approvals and required-construction-seal-approvals to 1 in the unstaked integration test

### CODE REVIEW

File: cmd/scaffold.go [hunk 1]
```diff
@@ -311,23 +311,25 @@ func (fnb *FlowNodeBuilder) initMetrics() {
...
-		Engine:     metrics.NewNoopCollector(),
-		Compliance: metrics.NewNoopCollector(),
-		Cache:      metrics.NewNoopCollector(),
-		Mempool:    metrics.NewNoopCollector(),
+		Network:        metrics.NewNoopCollector(),
+		Engine:         metrics.NewNoopCollector(),
+		Compliance:     metrics.NewNoopCollector(),
+		Cache:          metrics.NewNoopCollector(),
+		Mempool:        metrics.NewNoopCollector(),
+		CleanCollector: metrics.NewNoopCollector(),
```
Comment (huitseeker) [inline cmd/scaffold.go:319]: 👍

File: integration/tests/access/unstaked_node_test.go [hunk 2]
```diff
@@ -165,8 +165,8 @@ func (suite *UnstakedAccessSuite) buildNetworkConfig() {
 	consensusConfigs := []func(config *testnet.NodeConfig){
 		testnet.WithAdditionalFlag("--hotstuff-timeout=12s"),
 		testnet.WithAdditionalFlag("--block-rate-delay=100ms"),
-		testnet.WithAdditionalFlag(fmt.Sprintf("--required-verification-seal-approvals=%d", 0)),
-		testnet.WithAdditionalFlag(fmt.Sprintf("--required-construction-seal-approvals=%d", 0)),
+		testnet.WithAdditionalFlag(fmt.Sprintf("--required-verification-seal-approvals=%d", 1)),
+		testnet.WithAdditionalFlag(fmt.Sprintf("--required-construction-seal-approvals=%d", 1)),
```
Comment (huitseeker) [inline integration/tests/access/unstaked_node_test.go:169]: Are you sure? this seems pretty unrelated

**Review by smnzhu — APPROVED** [review]

**Review by huitseeker — APPROVED** [review]
